arch/sim/src/sim/posix/sim_hostfs.c

```diff
@@ -36,10 +36,66 @@
 
 #include "hostfs.h"
 
+#ifdef CONFIG_HOST_MACOS
+#include <sys/syslimits.h>
+#include <mach-o/dyld.h>
+#endif
+
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: host_path_convert
+ ****************************************************************************/
+
+static int host_path_convert(const char *path, char *abspath,
+                             uint32_t buflen)
+{
+#ifndef CONFIG_SIM_HOSTFS_PATH_CONVERSION
```

Comment:
```suggestion
#ifdef CONFIG_SIM_HOSTFS_PATH_CONVERSION
```

Reply:
It should be ifndef, only when the CONFIG_SIM_HOSTFS_PATH_CONVERSION option is enabled, the folder where nuttx is located will be used as the initial search path